client.go

```diff
@@ -185,18 +188,36 @@ func isEmpty(a any) bool {
 	return v.IsZero() || v.Kind() == reflect.Ptr && v.IsNil()
 }
 
-func fmtQuery(query ...[2]string) string {
+func fmtQuery(operationID, method string, query ...[2]string) string {
 	q := make(url.Values)
+
+	// Add all provided query parameters
 	for _, v := range query {
-		q.Add(v[0], v[1])
+		if v[0] != "" { // Skip empty keys
+			q.Add(v[0], v[1])
+		}
```

Comment:
Empty key is a valid value for query params.

Reply:
upd